Description: historyform: Resize columns to contents
 This patch backports commit c0d7e4ae89989d9882152cca4e539dec01d23f13.
Author: Peter Colberg <peter@colberg.org>
Forwarded: not-needed
Last-Update: 2015-12-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/gui/historyform.cpp
+++ b/src/gui/historyform.cpp
@@ -79,8 +79,11 @@ void HistoryForm::init()
     m_model->setHorizontalHeaderLabels(QStringList() << tr("Time") << tr("In/Out") << tr("From/To") << tr("Subject") << tr("Status"));
     historyListView->horizontalHeader()->setSortIndicator(HISTCOL_TIMESTAMP, Qt::DescendingOrder);
 
-    historyListView->setColumnWidth(HISTCOL_FROMTO, 200);
-    historyListView->setColumnWidth(HISTCOL_SUBJECT, 200);
+#if QT_VERSION >= 0x050000
+    historyListView->horizontalHeader()->setSectionResizeMode(QHeaderView::ResizeToContents);
+#else
+    historyListView->horizontalHeader()->setResizeMode(QHeaderView::ResizeToContents);
+#endif
 	
 	inCheckBox->setChecked(true);
 	outCheckBox->setChecked(true);
